content/raw/chat/group_chat.md

```diff
@@ -0,0 +1,50 @@
+# Group chat
+Group chats are meant to be private ad-hoc groups.
+
+# Rules
+
+### Glossary
+Group creator - one who creates new group  
+Member - anyone in the group  
+Admin - group administrator  
+User - Member who is not an Admin  
+
+## Group creation
+Group creator is only allowed to add mutual contacts. This is important, both for privacy and as a spam prevention measure.
+- Group chat MUST contain at least one Member
+- Group chat MUST be composed only of Group creator's mutual contacts
```

Comment:
Once a new admin is promoted, can new users be mutual contacts of the new admin but not mutual contacts of the first admin? Or do all users need to be mutual contacts of all admins?

Reply:
The specs have been updated.

- New group Member CAN be added by any Member
- Member CAN only add mutual contact to the group

So new member doesn't necessarily need to be mutual contact with every group member.